mutation_partition.cc

```diff
@@ -2029,6 +2030,17 @@ stop_iteration reconcilable_result_builder::consume_end_of_partition() {
     }
     _total_live_rows += _live_rows;
     _result.emplace_back(partition { _live_rows, _mutation_consumer->consume_end_of_stream() });
+
+    auto accounted = _memory_accounter.used_memory() - _used_at_entry;
+    auto actually_used = sizeof(partition) + _result.back().mut().representation().size();
+    if (actually_used > accounted) {
```

Comment:
Under what conditions do you expect this to be false?

Reply:
It should be true most of the time (that we under-account), but who can tell for sure. Hence the if.